tasks/base.py

```diff
@@ -159,6 +161,46 @@ def apply_async(self, args=None, kwargs=None, **options):
         }
         return super().apply_async(args=args, kwargs=kwargs, headers=headers, **options)
 
+    def _commit_django(self):
+        try:
+            django_transaction.commit()
+        except Exception as e:
+            log.warning(
+                "Django transaction failed to commit.",
+                exc_info=True,
+                extra=dict(e=e),
+            )
+
+        if timeseries_enabled():
```

Comment:
Wouldn't we be committing to timeseries a lot here for non-timeseries transactions if this is the condition? I read this as "if you have timeseries setup, and you come to this fn, you'll commit", but we commit to timeseries very infrequently so wouldn't we be overcommiting?

Reply:
https://github.com/codecov/worker/pull/283 will make base task log a few metrics to timescale for each task run, so we will need to commit the timeseries db

if we do away with the timescale metrics and just use alloy we could maybe skip the timeseries commit, but then again, committing an empty transaction is probably not expensive